packages/core/http/core-http-server-internal/src/http_server.ts

```diff
@@ -526,6 +527,10 @@ export class HttpServer {
       xsrfRequired: route.options.xsrfRequired ?? !isSafeMethod(route.method),
       access: route.options.access ?? (route.path.startsWith('/internal') ? 'internal' : 'public'),
     };
+    // Log HTTP API target consumer. Warning: may log sensitive information on paths including secrets
+    optionsLogger.debug(
+      `kibanaRouteOptions [${kibanaRouteOptions.access}] for path [${route.path}]`
```

Comment:
Shall we log the method too? IMO "access" is a clearer prefix than "kibanaRouteOptions":
```suggestion
      `access [${kibanaRouteOptions.access}] for path [${route.method} ${route.path}]`
```

Reply:
Approved!
I've also removed the code comment's warning